client/src/js/modules/types/saxs/dc/dc.js

```diff
@@ -1,24 +1,33 @@
 define([
     'modules/types/gen/dc/dc',
+    'modules/dc/views/apstatusitem',
+    'modules/dc/views/downstream',
     'modules/types/saxs/dc/datplot',
     'utils',
-    'templates/types/saxs/dc/dc.html'], function(DCItemView, DatPlot, utils, template) {
+    'templates/types/saxs/dc/dc.html'], function(DCItemView, APStatusItem, DCDownstreamView, DatPlot, utils, template) {
 
 
     return DCItemView.extend({
         template: template,
         plotView: DatPlot,
-        // imageStatusItem: DCImageStatusItem,
+        apStatusItem: APStatusItem,
         
         events: {
             'click .diffraction': 'showDiff',
             'click a.dd': utils.signHandler,
+            'click .holder h1.dp': 'loadAP',
         },
         
         showDiff: function(e) {
             e.preventDefault()
             this.$el.find('.diffraction a').eq(0).trigger('click')
         },
+
+        loadAP: function(e) {
+            if (!this.ap) {
+              this.ap = new DCDownstreamView({ id: this.model.get('ID'), el: this.$el.find('div.downstream') })
```

Comment:
This feels odd now. It's autoprocessing but using the downstream processing api. Presume this is because saxs uses processingjob as the link table rather than autoprocintegration. However, if saxs (or other types) extend to register jobs as both automated and downstream this gets confusing. Will this work scale to support detecting which jobs are downstream and which are reprocessed?

Reply:
Indeed this is just the way the database works. In ISPyB autoprocessing = integration and this is very mx specific. Any other technique will have to use "downstream". Its better that than renaming 15 tables. Theres no concept of "downstream" for other disciplines right now

Reply:
Reprocessed yes, but they will be reprocessed "downstream" for other techniques. The status ticks only show jobs with automatic = 1. But reprocessed will show in the tab, and we add a marker to show they are not automatic

Reply:
And for users they will obviously be marked as showsn as autoprocessing even if they use the "downstream" code